engine/Decompiler.Tests/Integration/MSBuildProjectBuilderTests.cs

```diff
@@ -16,11 +16,13 @@
 
 using Decompiler.Tests.Helpers;
 using JustDecompile.Tools.MSBuildProjectBuilder;
+using System;

```

Comment:
All AGPL-covered files should have their license (on the top of the file) updated for the current year.